Simplified/NYPLReadiumViewSyncManager.m

```diff
@@ -137,22 +137,43 @@ - (void)syncReadingPositionForBook:(NSString *)bookID
     NYPLLOG_F(@"serverLocationString %@",serverLocationString);
     NYPLLOG_F(@"currentLocationString %@",currentLocationString);
 
-    NSDictionary *spineItemDetails = weakSelf.bookMapDictionary[responseJSON[@"idref"]];
-    NSString *elementTitle = spineItemDetails[@"tocElementTitle"];
-    if (!elementTitle) {
-      elementTitle = @"";
-    }
-                
-    NSString *message = NSLocalizedString(@"Do you want to move to the page on which you left off?", nil);
-    NSAttributedString *atrString;
-    if (![elementTitle isEqualToString:@"Current Chapter"]) {
-      message = [message stringByAppendingString:[NSString stringWithFormat:@"<br><br>Chapter:\n&ldquo;%@&ldquo;", elementTitle]];
-      atrString = [[NSAttributedString alloc] initWithData:[message dataUsingEncoding:NSUTF8StringEncoding]
-                                                                       options:@{NSDocumentTypeDocumentAttribute: NSHTMLTextDocumentType,
-                                                                                 NSCharacterEncodingDocumentAttribute: @(NSUTF8StringEncoding)}
-                                                            documentAttributes:nil error:nil];
+    // Pass through without presenting the Alert Controller if:
+    // 1 - The most recent page on the server comes from the same device
+    // 2 - The server and the client have the same page marked
+    // 3 - There is no recent page saved on the server
+    if ((currentLocationString && [deviceIDString isEqualToString:[NYPLUserAccount sharedAccount].deviceID]) ||
+        [currentLocationString isEqualToString:serverLocationString] ||
+        !serverLocationString) {
+      weakSelf.shouldPostLastRead = YES;
+      return;
     }
+      
+    [self presentSyncReadingPositionAlertForResponse:responseJSON
```

Comment:
here I would use `weakSelf` for peace of mind... I'm not sure it's 100% needed, but the code before that is using it, so let's do the same.